ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -166,15 +171,36 @@ public final class AliasTokens: NSObject {
     }
     public let colors: TokenSet<ColorsTokens, DynamicColor>
 
+    // MARK: - Gradient Colors
+
+    @objc(MSFGradientColorAliasTokens)
+    public enum GradientTokens: Int, TokenSetKey {
+        case flair
+        case tint
+    }
+
+    @available(swift, obsoleted: 1.0, message: "This method exists for Objective-C backwards compatibility and should not be invoked from Swift. Please use the `gradientColors` property directly.")
+    @objc(aliasGradientForToken:)
```

Comment:
something about this objc naming doesn't sound like the return value will be array of colors

Reply:
Maybe we could change to `aliasGradientColorsForToken` in objc and `gradientColors` in swift?

Reply:
Sounds good, I'll do that